i18n/en/docusaurus-plugin-content-docs/current/component/09-selector.md

```diff
@@ -14,72 +14,70 @@ keywords:
 
 ```go
 type Selector interface {
-    // The list of service nodes maintained internally by the Selector is updated through the Rebalancer interface
+    // The list of service nodes maintained internally by the Selector is updated through the Rebalancer interface.
     Rebalancer
 
     // Select nodes
     // if err == nil, selected and done must not be empty.
     Select(ctx context.Context, opts ...SelectOption) (selected Node, done DoneFunc, err error)
 }
 
-// Realize service node change awareness through Rebalancer
+// Realize service node change awareness through Rebalancer.
 type Rebalancer interface {
     Apply(nodes []Node)
 }
 ```
+
 Supported implementations:
-* [wrr](https://github.com/go-kratos/kratos/tree/main/selector/wrr) : Weighted round robin (Kratos Client built-in default algorithm)
-* [p2c](https://github.com/go-kratos/kratos/tree/main/selector/p2c) : Power of two choices
-* [random](https://github.com/go-kratos/kratos/tree/main/selector/random) : Random
 
+- [wrr](https://github.com/go-kratos/kratos/tree/main/selector/wrr) : Weighted round robin (Kratos Client built-in default algorithm)
+- [p2c](https://github.com/go-kratos/kratos/tree/main/selector/p2c) : Power of two choices
+- [random](https://github.com/go-kratos/kratos/tree/main/selector/random) : Random
 
-### How to use
+## How to use
 
-#### HTTP Client
+### HTTP Client
 
 ```go
 import "github.com/go-kratos/kratos/v2/selector/p2c"
 import "github.com/go-kratos/kratos/v2/selector/filter"
 
-// Create a route Filter: filter instances with version number "2.0.0"
+// Create a route Filter: filter instances with version number "2.0.0".
 filter := filter.Version("2.0.0")
-// Create P2C load balancing algorithm Selector, and inject routing Filter
-selector := p2c.New(p2c.WithFilter(filter))
+// Create P2C load balancing algorithm Selector, and inject routing Filter.
+selctor.SetGlobalSelector(wrr.NewBuilder)
 
 hConn, err := http.NewClient(
-  context.Background(),
   http.WithEndpoint("discovery:///helloworld"),
   http.WithDiscovery(r),
-  // Inject Selector into HTTP Client through http.WithSelector
-  http.WithSelector(
-    p2c.New(p2c.WithFilter(filter.Version("2.0.0"))),
-  )
+  http.WithNodeFilter(filter)
 )
 ```
 
-#### gRPC Client
-
+### gRPC Client
 
 ```go
 import "github.com/go-kratos/kratos/v2/selector/p2c"
 import "github.com/go-kratos/kratos/v2/selector/filter"
 
-// Create a route Filter: filter instances with version number "2.0.0"
+// Create a route Filter: filter instances with version number "2.0.0".
 filter := filter.Version("2.0.0")
+// Due to the limitations of the gRPC framework, only the global balancer name can be used to inject Selector.
+selector.SetGlobalSelector(wrr.NewBuilder)
 
 conn, err := grpc.DialInsecure(
   context.Background(),
   grpc.WithEndpoint("discovery:///helloworld"),
```

Comment:
At the same time I have a question, The `///` in `discovery:///helloworld` is wrong or have  a meaning.  
I would appreciate it if  you told me this.

Reply:
you may refer to https://pkg.go.dev/google.golang.org/grpc/resolver#Target

Reply:
Thanks.

Reply:
> you may refer to https://pkg.go.dev/google.golang.org/grpc/resolver#Target

Thanks.